launch/tier4_localization_launch/launch/eagleye/fix2pose.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please make the file name's suffix ".launch.xml", unless there is a strong reason to avoid doing so.

Reply:
Fixed.